pkg/clusterd/context.go

```diff
@@ -57,4 +61,10 @@ type Context struct {
 
 	// The local devices detected on the node
 	Devices []*sys.LocalDisk
+
+	// CertManager webhook
+	Admission cs.CertmanagerV1Interface
```

Comment:
I don't see any tests that are mocking the code path. At least no unit tests are failing because this is not set. So I would propose that we not add these two fields to the Context struct. Let's just create these types when needed. It's just a one-time operation at operator startup and will never be re-used, right? So it doesn't seem necessary to pass to everyone in the context.

Reply:
refactored

Reply:
aah, I forgot to remove, removed now